packages/babel-plugin/src/css-prop/__tests__/index.test.tsx

```diff
@@ -1,14 +1,16 @@
 import { transformSync } from '@babel/core';
-import babelNext from '../../next';
+import babelPlugin from '../../index';
 
-jest.mock('@compiled/ts-transform-css-in-js/dist/utils/hash');
+jest.mock('@compiled/utils', () => {
+  return { ...jest.requireActual('@compiled/utils'), hash: () => 'hash-test' };
```

Comment:
because its a whole package now and not a single file we need to mock it like this. i couldnt get the `__mocks__` folder working - would take suggestions if you have any.

Reply:
Sometimes I feel manual mocks makes me jump between files and folders, which I really don't like. Mocking where your tests are gives me clear picture where things are coming from. I like being explicit than implicitly mocking things using manual mocks. It's like having a pure function with all variables defined inside it (or coming from arguments). I can be wrong here 🤔

Reply:
agree! for me it was a trade-off for writing the same mock every time vs. placing it in __mocks__ folder 😄

Reply:
Yups right 😉